### PR TITLE
ssh-agent: add page

### PR DESCRIPTION
**Pull request by nibalizer**
- [x ] The page (if new), does not already exist in the repo.
- [x ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x ] The page has 8 or fewer examples.
- [x ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/ssh-agent.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Kill the current agent
+
+`ssh-agent -k`
+
+- Manually connect to an existing agent (rare)
+
+`
+export SSH_AUTH_SOCK={{ssh-agent socket}}
+export SSH_AGENT_PID={{ssh-agent pid}}
```
Comment (zdroid) [inline pages/common/ssh-agent.md]: It should be one command per example.

Comment (nibalizer) [reply pages/common/ssh-agent.md]: I think you need to have both of these to get the effect you want.

Comment (sbrl) [reply pages/common/ssh-agent.md]: Indeed you do, @nibalizer. I think that the first example is sufficient though - but we could tweak the page description I guess.

File: pages/common/ssh-agent.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# ssh-agent
+
+> Spawn an ssh-agent process.
+> The ssh-agent will hold your ssh key decrypted in memory until evicted.
+> Add and manage keys with `ssh-add`.
```
Comment (sbrl) [inline pages/common/ssh-agent.md]: Perhaps this will read better?
```suggestion
> Spawn an SSH Agent process.
> An SSH Agent holds ssh keys decrypted in memory until removed or the process is killed.
> See also `ssh-add`, which can add and manage keys held by an SSH Agent.
```

File: pages/common/ssh-agent.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+- Start an SSH Agent for the current shell:
+
+`eval $(ssh-agent)`
+
+- Kill the current agent:
```
Comment (sbrl) [inline pages/common/ssh-agent.md]: Perhaps this will read better?
```suggestion
- Kill the currently running agent:
```

File: pages/common/ssh-agent.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+`ssh-agent -k`
+
+- Manually connect to an existing agent (rare):
+
+`export SSH_AUTH_SOCK={{ssh-agent socket}}`
+`export SSH_AGENT_PID={{ssh-agent pid}}`
```
Comment (sbrl) [inline pages/common/ssh-agent.md]: Just a suggestion per my previous comment.
```suggestion
`export SSH_AUTH_SOCK={{ssh_agent_socket}} SSH_AGENT_PID={{ssh_agent_pid}}`
```

Comment (nibalizer) [reply pages/common/ssh-agent.md]: We can do this if you want, I don't think exporting two variables on one line is clearer though.

Comment (sbrl) [reply pages/common/ssh-agent.md]: Sorry for aking so lonig to get back to you.

True, but what @zdroid was trying to say is that in tldr pages we only have 1 line of command for each example. In addition, I'm not really sure of the usefulness of this example as it is, since the `eval` one does this for you.

tldr pages aren't supposed to be _exhaustive_ - just include the most commonly used and most useful examples. So given the either existing example, I'm not sure we need this example unless you've got a good use-cas for not using the `eval` one.

Comment (nibalizer) [reply pages/common/ssh-agent.md]: Ok I will remove it. We can always add it back if that makes sense or a user asks for it!

**Comment by sbrl** [comment]
Hey! Next time, perhaps you could avoid force-pushing please? It makes it harder to tell what's changed when reviewing, and we've got a special button that allows us to squash easily on merge :smiley_cat:

**Comment by nibalizer** [comment]
@sbrl hey sorry - I won't force push next time. Thanks!
